layouts/partials/search-input.html

```diff
@@ -1,30 +1,82 @@
-{{ if or .Site.Params.gcs_engine_id .Site.Params.algolia_docsearch }}
-  <input
-    type="search"
-    class="form-control td-search-input"
-    placeholder="&#xf002; {{ T "ui_search_placeholder" }}"
-    aria-label="{{ T "ui_search_placeholder" }}"
-    autocomplete="off"
-  >
-{{ else if .Site.Params.offlineSearch }}
-  <div id="search-nav-container">
+<head>
```

Comment:
Why add this element?

Reply:
It was initially there in the code, I thought it may have some CSS properties attached to it